packages/library/src/commonMain/kotlin/io/realm/internal/Mediator.kt

```diff
@@ -16,14 +16,10 @@
 
 package io.realm.internal
 
-import io.realm.interop.Table
+import io.realm.RealmObject
 import kotlin.reflect.KClass
 
-// FIXME https://github.com/realm/realm-kotlin/issues/90 support default schema creation.
-interface Mediator { // avoid reflection, implemented and defined by compiler plugin for each `@RealmModule`
-    val companionMapping: Map<KClass<*>, RealmObjectCompanion>
-    fun newInstance(clazz: KClass<*>): RealmModelInternal
-    // This is constant so could just be computed directly in IR or at least just be a default method like
-    // fun schema(): List<Table> { return companionMapping.values.map { it.`$realm$schema`() } }
-    fun schema(): List<Table>
+interface Mediator {
```

Comment:
Maybe rename this to `SchemaMediator`? Right now the name is very generic and it is hard to guess what the responsibility is for this class.